.github/workflows/ci.yml

```diff
@@ -275,3 +275,61 @@ jobs:
           path: |
             docs/_build/html
             .pr_number
+
+  test_minimal_app:
+    name: Test Minimal Application with Base Dependencies
```

Comment:
Do we want to run these if `test` flow fails in general? maybe add a `requires: test` or something to these so that they arent wastefully ran if other issues are present? idk